lib/bolt/cli.rb

```diff
@@ -111,7 +111,10 @@ def parse
 
       validate(options)
 
-      @config = if options[:configfile]
+      @config = if !Bolt::Util.windows? && ENV['BOLT_PROJECT']
```

Comment:
Is there a reason not to support using the environment variable on Windows? Ostensibly the reason for this feature is to support using world writable directories, but it doesn't _have_ to be used for that and it could still be helpful on Windows.

Reply:
Heh, I was debating back and forth on that this morning and settled on not supporting it since I saw it as more closely tied with the world writable check. But I do think it would be generally helpful, so I'll remove the windows check.